src/applications/disability-benefits/view-payments/actions/index.js

```diff
@@ -30,7 +32,8 @@ export const getAllPayments = () => async dispatch => {
         'error-key': `${response.errors[0].status}_client_error`,
       });
     }
-    dispatch({ type: PAYMENTS_RECEIVED_FAILED, response });
+    const error = response?.errors?.[0];
```

Comment:
Line 23 already checks for `response.errors`, so I don't think we need these optional chaining checks here. Would it be better to move this `const error = ...` after line 23?